Package/PackageActions/SetUserToken.cs

```diff
@@ -0,0 +1,45 @@
+using System.ComponentModel;
+using System.Linq;
+using System.Threading;
+using OpenTap.Authentication;
+using OpenTap.Cli;
+namespace OpenTap.Package
+{
+    /// <summary> Sets the authentication token for use with a given domain.  </summary>
+    [Display("set-auth-token", Group: "package", Description: "Sets the authentication token for a given domain.")]
+    [Browsable(false)]
+    public class SetUserToken : ICliAction
```

Comment:
Can we call it `login` so it is a single verb like the other package actions? `create`, `list` etc.